beacon-chain/rpc/eventsv1/events.go

```diff
@@ -0,0 +1,189 @@
+package eventsv1
+
+import (
+	gwpb "github.com/grpc-ecosystem/grpc-gateway/v2/proto/gateway"
+	"github.com/pkg/errors"
+	"github.com/prysmaticlabs/prysm/beacon-chain/core/feed"
+	blockfeed "github.com/prysmaticlabs/prysm/beacon-chain/core/feed/block"
+	"github.com/prysmaticlabs/prysm/beacon-chain/core/feed/operation"
+	statefeed "github.com/prysmaticlabs/prysm/beacon-chain/core/feed/state"
+	ethpb "github.com/prysmaticlabs/prysm/proto/eth/v1"
+	"github.com/prysmaticlabs/prysm/proto/migration"
+	log "github.com/sirupsen/logrus"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/types/known/anypb"
+)
+
+var casesHandled = map[string]bool{
+	"head":                 true,
+	"block":                true,
+	"attestation":          true,
+	"voluntary_exit":       true,
+	"finalized_checkpoint": true,
+	"chain_reorg":          true,
```

Comment:
Can you extract these names to variables? Each is used at least 3 times in this file and it will be more in tests, I presume.